<testsuites name="results">
  <testsuite name="all" package="all">
    <property name="random_seed" value="1753219400" />
    <testcase name="test_add" classname="unit.alu.verify" file="/home/ckashino/programming/verilog/8bit/sim/unit/alu/verify.py" lineno="12" time="0.00043582916259765625" sim_time_ns="0.011" ratio_time="25.23924726477024">
      <failure message="Test failed with RANDOM_SEED=1753219400" />
    </testcase>
    <testcase name="test_add_carry_out" classname="unit.alu.verify" file="/home/ckashino/programming/verilog/8bit/sim/unit/alu/verify.py" lineno="25" time="0.00018262863159179688" sim_time_ns="0.011" ratio_time="60.231519582245426" />
    <testcase name="test_add_carry_in" classname="unit.alu.verify" file="/home/ckashino/programming/verilog/8bit/sim/unit/alu/verify.py" lineno="40" time="0.00011157989501953125" sim_time_ns="0.011000000000000003" ratio_time="98.5840683760684" />
    <testcase name="test_sub" classname="unit.alu.verify" file="/home/ckashino/programming/verilog/8bit/sim/unit/alu/verify.py" lineno="55" time="0.00011372566223144531" sim_time_ns="0.010999999999999996" ratio_time="96.72399161425572" />
    <testcase name="test_sub_neg_arg" classname="unit.alu.verify" file="/home/ckashino/programming/verilog/8bit/sim/unit/alu/verify.py" lineno="69" time="0.0001125335693359375" sim_time_ns="0.011000000000000003" ratio_time="97.74861016949156" />
    <testcase name="test_sub_neg_result" classname="unit.alu.verify" file="/home/ckashino/programming/verilog/8bit/sim/unit/alu/verify.py" lineno="85" time="0.00010275840759277344" sim_time_ns="0.011000000000000003" ratio_time="107.04720185614852" />
  </testsuite>
</testsuites>
